main.tf

```diff
@@ -1,10 +1,11 @@
 locals {
   APP_NAME = "workadventure-jitsi"
   APP_HOST = "jitsi.dev.dorf.world"
-  DOCKER_HOST_ADDRESS = null # Set this to the Internet routable IPv4 address of the node?
+  PUBLIC_IP_ADDRESS = "45.151.166.29"
```

Comment:
please use a Terraform variable for the IP address

Reply:
That _is_ the variable. Did you mean to comment another line?

Reply:
My suggestion is to put the value of the IP address into a Terraform variable, rather than setting it here. In the Terraform Cloud workspace, you can then set the variable. This way, we can have multiple instances of our setup (in multiple workspaces, using different values for this variable).